tests/testproject/TestProject.AppHost/TestProgram.cs

```diff
@@ -65,7 +65,7 @@ public class TestProgram : IDisposable
         {
             // Relative to this project so that it doesn't changed based on
             // where this code is referenced from.
-            var path = Path.Combine(Projects.TestProject_AppHost.ProjectPath, @"..\nodeapp");
+            var path = Path.Combine(Projects.TestProject_AppHost.ProjectPath, "..", "nodeapp");
```

Comment:
If the nodeapp fails to start for some reason, then no error is being surfaced. @ReubenBond